Packs/Zendesk/Playbooks/playbook-Zendesk_-_Ticket_Management.yml

```diff
@@ -0,0 +1,830 @@
+id: Zendesk - Ticket Management
+version: -1
+name: Zendesk - Ticket Management
+description: '`Zendesk - Ticket Management` allows you to open new tickets or update comments to the existing ticket using the playbook''s tasks. '
```

Comment:
"using the playbook's tasks.' is needed here?
"..a new ticket..or comment on an existing ticket."

Reply:
Done.